### PR TITLE
Typo fixes

### PR DESCRIPTION
**Pull request by luzpaz**
Found via `codespell -q 3 -S *.js -L ba`

### CODE REVIEW

File: cadquery/cq.py [hunk 1]
```diff
@@ -698,7 +698,7 @@ def first(self: T) -> T:
     def item(self: T, i: int) -> T:
         """
 
-        Return the ith item on the stack.
+        Return the nth item on the stack.
```
Comment (jmwright) [inline cadquery/cq.py:701]: I think `ith` was correct here since the index variable was `i`.

**Review by jmwright — APPROVED** [review]
Thanks @luzpaz this is great. I just had the one comment.

I wonder if the check you did should be added to CI to prevent these types of spelling errors from creeping back in.

File: cadquery/cq.py (hunk 1 above)
Comment (luzpaz) [reply cadquery/cq.py]: Sorry about that, reverted in a7f62f2

**Comment by luzpaz** [comment]
> I wonder if the check you did should be added to CI to prevent these types of spelling errors from creeping back in.

Yes, some repos have done this. I don't have that info in front of me right now.

**Comment by jmwright** [comment]
@marcus7070 @adam-urbanczyk Do either of you want to check these spelling fixes before I merge?

**Comment by luzpaz** [comment]
> > I wonder if the check you did should be added to CI to prevent these types of spelling errors from creeping back in.
> 
> Yes, some repos have done this. I don't have that info in front of me right now.

Check out https://github.com/codespell-project/codespell/issues/1499

**Comment by adam-urbanczyk** [comment]
I skimmed and it LGTM.

**Review by adam-urbanczyk — APPROVED** [review]

**Comment by jmwright** [comment]
@luzpaz Thanks!

**Pull request merged**